tap_aggregator/src/aggregator.rs

```diff
@@ -194,71 +191,65 @@ mod tests {
     #[rstest]
     #[tokio::test]
     /// Test that a receipt with a timestamp greater then the rav timestamp passes
-    async fn check_receipt_timestamps_ok(
-        keys: (LocalWallet, Address),
-        allocation_ids: Vec<Address>,
-    ) {
-        let time = SystemTime::now()
-            .duration_since(UNIX_EPOCH)
-            .unwrap()
-            .as_millis() as u64;
+    async fn check_receipt_timestamps(keys: (LocalWallet, Address), allocation_ids: Vec<Address>) {
+        // Create receipts with consecutive timestamps
+        let mut receipts = Vec::new();
+        for i in 10..20 {
```

Comment:
It seems these numbers are used throughout the test. Could this be updated be more clear that the numbers are all related? Possible store this range in a variable and use `max()` / `min()` on the range where applicable.

Reply:
Fixed in [0eca077](https://github.com/semiotic-ai/timeline-aggregation-protocol/pull/116/commits/0eca077705b505b604b612d7a8c0dc64dc1baafd)